core/arch/x86_64/kernel/boot.c

```diff
@@ -1201,13 +606,9 @@ static void init_primary(unsigned long pageable_part, unsigned long nsec_entry)
  * Note: this function is weak just to make it possible to exclude it from
  * the unpaged area.
  */
-void __weak boot_init_primary_late(unsigned long fdt)
+void __weak boot_init_primary_late(unsigned long fdt __unused)
```

Comment:
Why we don't remove fdt parameter directly?

Reply:
It is also defined in link_dummies_paged.c, I dont't want to modify that file